ntbs-service/Data/Legacy/SearchService.cs

```diff
@@ -5,19 +5,19 @@
 namespace ntbs_service.Data.Legacy
 {
 
-    public interface ISearchService
+    public interface ISearchServiceLegacy

```

Comment:
AFAIK we want to keep this code so have renamed this service and kept it in the codebase, correct me if I'm wrong @jasiekmiko!